### PR TITLE
import into: change max pool size to num-cpu

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #42930 

Problem Summary:

### What is changed and how it works?
- change max pool size to num-cpu, so import be able to use all cpu resource, previously it's 10 independently on TiDB cpu count

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
  - after change cfg, import is 8(8c on local mac)
nTF41gZrQL
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]
lgtm, however we might need to check some resources should be left for other purposes

File: disttask/importinto/scheduler.go [hunk 1]
```diff
@@ -223,7 +224,7 @@ func init() {
 		logger.Info("create step scheduler")
 		return &taskMeta, logger, nil
 	}
-	scheduler.RegisterTaskType(proto.ImportInto)
+	scheduler.RegisterTaskType(proto.ImportInto, scheduler.WithPoolSize(int32(runtime.NumCPU())))
```
Comment (hawkingrei) [inline disttask/importinto/scheduler.go:227]: Please not use ```runtime.NumCPU()```. If it is in the container. it will get wrong value.

**Comment by D3Hunter** [comment]
changed to GETMAXPROCS, see https://stackoverflow.com/a/61866154 also

**Review by hawkingrei — APPROVED** [review]

**Comment by hawkingrei** [comment]
/hold

need more people to review.

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**